src/Microsoft.AspNet.Mvc.Core/ModelBinding/CompositeModelBinder.cs

```diff
@@ -195,6 +169,32 @@ public CompositeModelBinder([NotNull] IEnumerable<IModelBinder> modelBinders)
                 }
             }
```

Comment:
Can we now move this early-exit block above creation of `newBindingContext`? Doesn't depend on `BodyBindingState`.